### PR TITLE
feat: [ANDROAPP-3919] Flow Form 

### PR DESCRIPTION
**Pull request by JaimeToca**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://jira.dhis2.org/browse/

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: form/src/main/java/org/dhis2/form/ui/RecyclerViewUiEvents.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        val uid: String,
+        val year: Int,
+        val month: Int,
+        val day: Int
+    ) : RecyclerViewUiEvents()
+
+    data class OpenCustomAgeCalendar(val uid: String, val label: String) : RecyclerViewUiEvents()
+    data class ShowDescriptionLabelDialog(val title: String, val message: String?) :
+        RecyclerViewUiEvents()
+}
```
Comment (JaimeToca) [inline form/src/main/java/org/dhis2/form/ui/RecyclerViewUiEvents.kt:14]: @ferdyrod what do you think?

**Review by mmmateos — APPROVED** [review]

File: form/src/main/java/org/dhis2/form/mvi/MviIntent.kt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+package org.dhis2.form.mvi
+
+interface MviIntent
```
Comment (ferdyrod) [inline form/src/main/java/org/dhis2/form/mvi/MviIntent.kt:3]: @JaimeToca What will this empty interface do in the future?

File: form/src/main/java/org/dhis2/form/ui/RecyclerViewUiEvents.kt (hunk 1 above)
Comment (ferdyrod) [reply form/src/main/java/org/dhis2/form/ui/RecyclerViewUiEvents.kt:14]: This is great! 👍

**Review by ferdyrod — APPROVED** [review]
Awesome job!

**Pull request merged**